src/org/gk/qualityCheck/HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck.java

```diff
@@ -0,0 +1,100 @@
+package org.gk.qualityCheck;
+
+import org.gk.database.InstanceListPane;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+/**
+ * Flags all human Reactions that contain Non-only-Human PhysicalEntities that do not have a populated disease attribute.
+ * Valid PhysicalEntities are those that have a non-human species or that have a human species AND have a filled relatedSpecies attribute.
+ */
+public class HumanReactionsWithNonHumanPhysicalEntitiesWithoutDiseaseCheck extends AbstractQualityCheck {
+
+    @Override
+    public QAReport checkInCommand() throws Exception {
+        QAReport report = new QAReport();
+        MySQLAdaptor dba = (MySQLAdaptor) dataSource;
+        QACheckUtilities.setHumanSpeciesInst(dba);
+        QACheckUtilities.setSkipList(Files.readAllLines(Paths.get("QA_SkipList/Manually_Curated_NonHuman_Pathways.txt")));
```

Comment:
Could this be broken into multiple lines for readability 

```
QACheckUtilities.setSkipList(
    Files.readAllLines(
        Paths.get("QA_SkipList/Manually_Curated_NonHuman_Pathways.txt")
    )
);
```

or maybe create and call a simple helper method named something like "getNonHumanPathwaySkipList()"

`QACheckUtilities.setSkipList(getNonHumanPathwaySkipList());`

Reply:
Personally, I would avoid the multi-line version. It _looks_ like multiple expressions separated with `{` and `}` (because of the way it's indented), but it really is evaluated as one expression. Also, breaking up code like that makes me twitchy. Better to just have a few variable assignments. But this is a personal style thing, so... :shrug:

Reply:
I will implement `QACheckUtilities.setSkipList(getNonHumanPathwaySkipList());`